dist/networked-aframe.js

```diff
@@ -2171,6 +2181,11 @@
 	    if (this.isMine() && NAF.connection.isConnected()) {
 	      var syncData = { networkId: this.data.networkId };
 	      NAF.connection.broadcastDataGuaranteed('r', syncData);
+	      if (NAF.entities.hasEntity(this.data.networkId)) {
+	        delete NAF.entities.entities[this.data.networkId];
```

Comment:
This data structure belongs to NetworkEntities, so this should be a call to it to clean it up